clickstream-event-visualiser-ui/src/main/java/clickstream/eventvisualiser/ui/internal/data/repository/CSEvRepository.kt

```diff
@@ -1,9 +1,12 @@
 package clickstream.eventvisualiser.ui.internal.data.repository
 
 import clickstream.eventvisualiser.ui.internal.data.model.CSEvEvent
+import kotlinx.coroutines.flow.Flow
 
 internal interface CSEvRepository {
 
+    val connectionFlow: Flow<Boolean>
```

Comment:
similar to https://github.com/gojek/clickstream-android/pull/25/files#r952239772